concent_api/core/views.py

```diff
@@ -530,6 +539,160 @@ def handle_send_force_subtask_results_results_response(request, client_message):
     return HttpResponse("", status = 202)
 
 
+def verify_message_subtask_results_accepted(subtask_results_accepted_list):
+    verify_public_key           = len(set(subtask_results_accepted.task_to_compute.requestor_public_key             for subtask_results_accepted in subtask_results_accepted_list)) == 1
+    verify_ethereum_public_key  = len(set(subtask_results_accepted.task_to_compute.requestor_ethereum_public_key    for subtask_results_accepted in subtask_results_accepted_list)) == 1
+    return bool(verify_public_key is True and verify_ethereum_public_key is True)
+
+
+def sum_payments(list_of_payments):
+    return sum([item.price for item in list_of_payments])
+
+
+def handle_send_force_payment(request, client_message):
+    client_public_key       = decode_client_public_key(request)
+    other_party_public_key  = decode_other_party_public_key(request)
+    current_time            = get_current_utc_timestamp()
+
+    if not verify_message_subtask_results_accepted(client_message.subtask_results_accepted_list):
+        return message.concents.ServiceRefused(
+            reason = message.concents.ServiceRefused.REASON.InvalidRequest
+        )
+    requestor_ethereum_public_key = client_message.subtask_results_accepted_list[0].task_to_compute.requestor_ethereum_public_key
+
+    # Concent defines time T0 equal to oldest payment_ts from passed SubtaskResultAccepted messages from subtask_results_accepted_list.
+    T0 = min(subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list)
```

Comment:
In the final version please rename `T0`, `T1` and `V` to more meaningful names, so everyone can know what they are without checking blueprints.